### PR TITLE
Integration test: basic cluster

### PR DESCRIPTION
**Pull request by tock-ibm**

### CODE REVIEW

File: test/setup/config_writer.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+// Provide full path with .yaml suffix.
+func WriteLocalConfig(localConfig *config.LocalConfiguration, localConfigYamlFile string) error {
+	return writeConfigAsYaml(localConfig, localConfigYamlFile)
+}
+
+func WriteSharedConfig(sharedConfig *config.SharedConfiguration, sharedConfigYamlFile string) error {
+	return writeConfigAsYaml(sharedConfig, sharedConfigYamlFile)
+}
+
+func writeConfigAsYaml(configObject interface{}, configYamlFile string) error {
```
Comment (gennadylaventman) [inline test/setup/config_writer.go:21]: Didn't we agree to copy code from sdk?

```
func WriteConfigToYaml(configObject interface{}, configFile string) error {
	data, err := yaml.Marshal(configObject)
	if err != nil {
		return err
	}
	err = ioutil.WriteFile(configFile, data, 0644)
	if err != nil {
		return err
	}
	return nil
}
```

File: internal/httputils/utils.go [hunk 2]
```diff
@@ -26,6 +26,15 @@ func MarshalOrPanic(m proto.Message) []byte {
 	return bytes
 }
 
+func MarshalJsonOrPanic(o interface{}) []byte {
```
[the remark this answers is not in the document]
Comment (tock-ibm) [reply internal/httputils/utils.go:29]: https://github.com/hyperledger-labs/orion-sdk-go/issues/77

Done

File: test/setup/config_writer.go (hunk 1 above)
Comment (tock-ibm) [reply test/setup/config_writer.go:21]: Done

File: test/cluster/basic_test.go [hunk 3]
```diff
@@ -0,0 +1,103 @@
+// Copyright IBM Corp. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cluster
+
+import (
+	"fmt"
+	"github.com/pkg/errors"
```
[the remark this answers is not in the document]
Comment (tock-ibm) [reply test/cluster/basic_test.go:8]: done

File: test/setup/config_writer.go [hunk 4]
```diff
@@ -0,0 +1,41 @@
+package setup
+
+import (
+	"github.com/hyperledger-labs/orion-server/config"
```
[the remark this answers is not in the document]
Comment (tock-ibm) [reply test/setup/config_writer.go:7]: done

**Review by gennadylaventman — DISMISSED** [review]

**Review by gennadylaventman — APPROVED** [review]

**Pull request merged**